src/daemon/methods.rs

```diff
@@ -9,6 +9,7 @@ use dbus::{
 };
 
 use crate::misc::format_error;
+use num_derive::FromPrimitive;
 use num_traits::FromPrimitive;
```

Comment:
Are both imports required?

Reply:
yep. though i agree it looks weird. also surprising that they don't collide (they don't)

Reply:
> also surprising that they don't collide

Macros are in a different namespace from types, which I guess is both a legacy from the pre-2018 way of importing macros, and because a derive macro is typically the same name as the trait it derives.